### PR TITLE
[cb-fetch_v1.11.x] update cb-fetch libdef

### PR DESCRIPTION
**Pull request by Mouvedia**
Links to documentation: https://github.com/Mouvedia/cb-fetch/blob/master/README.md
Link to GitHub or NPM: https://github.com/Mouvedia/cb-fetch
Type of contribution: new definition (update)

Other notes:

I am gonna release 1.11.0 soonish and I want the libdef to be ready by then.

Related:

#3096

### CODE REVIEW

File: definitions/npm/cb-fetch_v1.11.x/flow_v0.104.x-/cb-fetch_v1.11.x.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+declare module 'cb-fetch' {
+
+    declare module.exports: typeof request;
+
+    declare type RequestBody =
+      | BodyInit
```
Comment (pascalduez) [inline definitions/npm/cb-fetch_v1.11.x/flow_v0.104.x-/cb-fetch_v1.11.x.js]: `BodyInit` doesn't seem to be declared in the `104.x` def file.

Comment (Mouvedia) [reply definitions/npm/cb-fetch_v1.11.x/flow_v0.104.x-/cb-fetch_v1.11.x.js]: It's declared [here](https://github.com/facebook/flow/blob/master/lib/bom.js#L1398).

File: definitions/npm/cb-fetch_v1.11.x/flow_v0.37.x-v0.103.x/cb-fetch_v1.11.x.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
+declare module 'cb-fetch' {
+
+    declare module.exports: typeof request;
+
+    declare type BodyInit = string | URLSearchParams | FormData | Blob | ArrayBuffer | $ArrayBufferView | ReadableStream
```
Comment (pascalduez) [inline definitions/npm/cb-fetch_v1.11.x/flow_v0.37.x-v0.103.x/cb-fetch_v1.11.x.js]: Why the override here? It's not done in the `104.x` def file, and seems to be in Flow core for a long time.

**Comment by pascalduez** [comment]
You know you could also ship an `index.js.flow` along the package? You already have the TS one.

File: definitions/npm/cb-fetch_v1.11.x/flow_v0.37.x-v0.103.x/cb-fetch_v1.11.x.js (hunk 2 above)
Comment (Mouvedia) [reply definitions/npm/cb-fetch_v1.11.x/flow_v0.37.x-v0.103.x/cb-fetch_v1.11.x.js]: Because it is not supported by 0.37–0.41.

**Pull request merged**